generators/_init-web/templates/plain/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  root: true,
+  extends: [
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "plugin:prettier/recommended"
+  ],
+  plugins: ["react", "@typescript-eslint", "jest"],
+  ignorePatterns: ["*.js"],
+  env: {
+    browser: true,
+    es6: true,
+    jest: true
+  },
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly",
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    // Make sure eslint and VS Code use the same path for the tsconfig.json:
+    // https://github.com/typescript-eslint/typescript-eslint/issues/251
+    tsconfigRootDir: __dirname,
+    project: "./tsconfig.eslint.json"
+  },
+  rules: {
+    // Disables jsx-a11y https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-webpack-loader-syntax.md
+    ...Object.keys(require('eslint-plugin-jsx-a11y').rules).reduce((acc, rule) => { acc[`jsx-a11y/${rule}`] = 'off'; return acc }, {}),
+    "class-methods-use-this":"off",
+    "linebreak-style": "off",
+    "no-continue": "off",
+    "no-multi-assign": "warn",
+    "no-nested-ternary": "off",
+    "no-param-reassign": ["error", { "props": false }],
```

Comment:
You define `no-param-reassign` here, but override it below. Remove one of these config lines and change according to the suggestion below.